docs/commands/ft.aggregate.md

```diff
@@ -4,7 +4,7 @@ Non-deterministic. Depends on the query and aggregations performed, but it is us
 
 ---
 
-Runs a search query on an index, and performs aggregate transformations on the results, extracting statistics etc from them. See [the full documentation on aggregations](/redisearch/reference/aggregations) for further details.
+Runs a search query on an index, and performs aggregate transformations on the results, extracting statistics etc from them. See [the full documentation on aggregations](/docs/stack/search/reference/aggregations) for further details.
```

Comment:
@itamarhaber Shouldn't links be handled during web content generation (to be mapped to the root of the module content)?